selfdrive/ui/paint.cc

```diff
@@ -213,20 +205,12 @@ static void update_all_track_data(UIState *s) {
 
 
 static void ui_draw_track(UIState *s, bool is_mpc, track_vertices_data *pvd) {
+ if (pvd->cnt == 0) return;
+
   nvgBeginPath(s->vg);
-  bool started = false;
-  for(int i = 0;i < pvd->cnt;i++) {
-    float x = pvd->v[i].x;
-    float y = pvd->v[i].y;
-    if (x < 0 || y < 0) {
```

Comment:
x,y always >=0

Reply:
how did you test that? I'm fine with just merging the change to started.

Reply:
function update_lane_line_date has already call valid_frame_pt to make sure x,y>=0

Reply:
makes sense